src/components/va-accordion/va-accordion.tsx

```diff
@@ -27,7 +27,7 @@ export class VaAccordion {
 
   @Listen('accordionItemToggled')
   itemToggledHandler(event: CustomEvent) {
-    const clickedItem = event.detail.target.parentNode.host;
+    const clickedItem = event.detail.target.parentNode.parentNode.host;
```

Comment:
Now that the `<button>` is wrapped in a `<hN>`, we have an extra parent to deal with.

Reply:
Oof. This is getting gnarly enough I'd love to see a comment on it describing what exactly this is grabbing.